independent-projects/arc/runtime/src/main/java/io/quarkus/arc/Subclass.java

```diff
@@ -7,8 +7,4 @@
  */
 public interface Subclass {
```

Comment:
With the method removed from here, this interface is now empty - does it make sense to keep it just as a marker interface?

Reply:
I believe it could be useful ;-)

Reply:
Hmm, I suppose you are right, after all, similar thing is in Weld for subclasses and for proxies...

Reply:
Yes, please keep it, I use it :).